### DISCUSSION TITLE
#NowPlaying

### DISCUSSION THREAD

**Discussion by richardbirkin** [SuperHyperInstantFutureTime/TrackShift#159] [General]
We listen to some amazing tunes while we're working together at Greg's house. This thread is to shout to each other about what we're listening to when we're working remotely.

**Comment by richardbirkin** [comment]
@g105b 

[The Closest Thing To Silence](https://open.spotify.com/album/3J4TcjEkyeAj5XB0Y3E1We?si=DFJcY9aJQEqN9Vk0Lu-FJQ) 
by Ariel Kalma, Jeremiah Chiu, Marta Sofia Honer 
released by International Anthem

**Reply by g105b** [reply]
Nice idea. I'll queue it up! 

This absolute old timer classic is playing currently and enlightening my ears. One of my favourite audio delights of all time! Lots of minimoog and other lovely synthy things.

[Wish You Were Here](https://open.spotify.com/album/6uvBKDGlJAYLH5Vy6RQVsc?si=q9nEjGwAT4m37OHQcSyZrA)
by Pink Floyd

**Comment by richardbirkin** [comment]
[Imagine The Future](https://yunx.bandcamp.com/album/imagine-the-future)
by Yunx

I met Iain Law while working for a client, and we shared our musics with each other. This album is great. Squelchy bubbly electronica.

**Comment by richardbirkin** [comment]
[Seasons Change](https://imagine-the-sea.bandcamp.com/album/seasons-change)
by Imagine The Sea

Greg was playing this at his house after lunch as we discussed adding filters to TS for smoother rendering of money values

**Comment by g105b** [comment]
[Kid Velo](https://rivalconsoles.bandcamp.com/album/kid-velo)
by Rival Consoles

Juicy atmospheric synths!
